js/mobile_menu.js

```diff
@@ -13,4 +13,231 @@ function menuPop() {
   }
 }
 
-menuPop();
\ No newline at end of file
+const worksContent = [
+  {
+    id: 1,
+    worksName: 'Data Dashboard Healthcare',
+    worksContent:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.",
+    techStack: ['HTML', 'Bootstrap', 'Ruby'],
+    img: 'images/icons/pro_sec.svg',
+  },
```

Comment:
- [ ] Kindly make sure to place two properties for the link to the live version, and the link to the project this should be applied to each project object and should contain their respective links. (This can be a dummy link for now 🤓).